Fix EBmol

esclapez opened the pull request:
Allow EB+MOL to run with the latest IAMR developments.

esclapez committed:
Add EBgodunov and Redistribution from IAMR when using EB.

esclapez committed:
Update calls to MOL::ComputeAofS.

esclapez committed:
MOL is now called for non-EB in NSB -> take it out of the #ifdef EB
statement.

ldowen commented on Tools/Make/Make.PeleLM:
```diff
@@ -103,6 +103,8 @@ Bdirs += $(IAMR_HOME)/Source/Src_$(DIM)d
 Bdirs += $(IAMR_HOME)/Source/Godunov
 ifeq ($(USE_EB), TRUE)
    Bdirs   += $(IAMR_HOME)/Source/MOL
```
I think this line needs to be moved outside the EB check since certain files are referenced in NavierStokesBase.cpp no matter what.

esclapez replied:
That seems to be a weirdness on the IAMR side then. I'll try to fix that.

esclapez replied:
Hum ... MOL isn't split between EB and non-EB so there might not be a way around it.

ldowen replied:
My code worked fine when I moved it out of the if statement.

esclapez replied:
Indeed, they are now using non-EB MOL in NSB.

esclapez committed:
ComputeAofS new returns 0.0 on covered edges.
It turns out we where using covered egde value (previously at 1e40).
Bypass computing rhoHm in the covered ghost faces.

esclapez committed:
Update IAMR and amrex Submodules

esclapez committed:
Adding CMake option.

jrood-nrel committed:
Update CMake for latest IAMR submodule.

esclapez committed:
Update IAMR submodule.

esclapez commented:
Thanks @jrood-nrel  @ldowen for the help. Ready to merge.
